app/controllers/admin/programs_controller.rb

```diff
@@ -17,8 +17,9 @@ def update
         ConferenceScheduleUpdateMailJob.perform_later(@conference) if send_mail_on_schedule_public
         respond_to do |format|
           format.html do
-            redirect_to admin_conference_program_path(@conference.short_title),
-                        notice: 'The program was successfully updated.'
+            notice = "The program was successfully updated."
+            notice += " You changed schedule interval and schedule can be changed. Please check." if @program.previous_changes.has_key?(:schedule_interval)
```

Comment:
what about showing this only if an event_schedule was destroyed?

And then the message can be:

_You changed schedule interval and some events were unscheduled_